.github/workflows/publish-chains-config.yaml

```diff
@@ -0,0 +1,25 @@
+name: 'Publish axelar-chains-config to NPM'
+
+on:
+    workflow_dispatch:
+    release:
+        types: [published]
```

Comment:
this will be published whenever a release for the repo is created. if there are no changes to this package, it'll be pushed with the same version?

Reply:
the action will be run whenever the release is created, however, the `npm publish` will be simply failed if the version in package.json is not updated.

Reply:
ok, it'll be a bit weird since we might release/update deployment scripts package more often than chain configs which will cause failed runs. shall we make it manually run instead on publish?